src/components/HostLayout.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { NavLink, Link, Outlet } from "react-router-dom";
+
+const HostLayout = () => {
+  return (
+    <>
```

Comment:
You don't need this Fragment because fragments are used when there is no parent tag, but in your case, you already have one, which means you don't need it.